data/base-crafting.yaml

```diff
@@ -1137,7 +1137,7 @@ artificing:
     - 15525
     - 15527
     - 15528
-    tool-room: 15522
+    tool-room: 1556
```

Comment:
needs fixed

Reply:
@robbintt are you saying "needs fixed" as a reason for the line change from room `15522` => `1556` or did you realize that room `1556` is not in Hibarnhvidar and so it needs to be fixed from what this PR is suggesting (as in, there's another update to be made to correct the room number)?

Reply:
+1

Reply:
looks like a typo from me